### PR TITLE
Adding Changelog for PATCH Chat Message Validation Update

### PR DESCRIPTION
**Pull request by evanypark**
Changelog for PATCH Chat Message will now require message sender to be part of same tenant as request originator, hence blocking PATCH operation on federated chat message.

### CODE REVIEW

File: changelog/Microsoft.Teams.Core.json [hunk 1]
```diff
@@ -1,5 +1,23 @@
 {
   "changelog": [
+    {
+      "ChangeList": [
+        {
+          "Id": "04f199ca-74fe-4c67-b39f-2feacad3f304",
+          "ApiChange": "Authentication",
+          "ChangedApiName": "Messaging",
+          "ChangeType": "Change",
+          "Description": "Enabled validation on chat message sender tenant id, when performing PATCH operation on chat message.",
```
Comment (RamjotSingh) [inline changelog/Microsoft.Teams.Core.json:10]: Add

After this change, only tenant whose user sent the message, will be able to perform Data Loss Prevention PATCH on the chat message i.e. change the `policyViolation` property on a message in a chat.

Add links for chat and message in the text above.

File: api-reference/beta/api/chatmessage-update.md [hunk 2]
```diff
@@ -30,6 +30,11 @@ One of the following permissions is required to call this API. To learn more, in
 |Delegated (personal Microsoft account) | Not supported.    |
 |Application | Chat.UpdatePolicyViolation.All for a chat message.</br>ChannelMessage.UpdatePolicyViolation.All for a channel message. |
 
+## Federation
```
Comment (Lauragra) [inline api-reference/beta/api/chatmessage-update.md:33]: Please do not add custom H2 headings to topics. For consistency, all API topics should include the standard H2 headings.

I suggest that you add this as regular text to the summary section of the topic. Suggested revision:

This method does not support federation. Only the user in the tenant who sent the message can perform data loss prevention (DLP) updates on the specified chat message.

File: changelog/Microsoft.Teams.Core.json [hunk 3]
```diff
@@ -1,5 +1,23 @@
 {
   "changelog": [
+    {
+      "ChangeList": [
+        {
+          "Id": "04f199ca-74fe-4c67-b39f-2feacad3f304",
+          "ApiChange": "Authentication",
```
Comment (Lauragra) [inline changelog/Microsoft.Teams.Core.json:7]: Is this an API-level change? If not, it doesn't warrant a changelog entry. If so, please revise as follows:

Enabled validation on the sender tenant ID for the [Update chatMessage](https://learn.microsoft.com/en-us/graph/api/update-chatmessage) operation. With this change, only the tenant of the user who sent the message can perform data loss prevention updates on the chat message; that is, change the **policyViolation** property on the chat message.

File: api-reference/beta/api/chatmessage-update.md (hunk 2 above)
Comment (evanypark) [reply api-reference/beta/api/chatmessage-update.md]: Change has been updated.

**Review by Lauragra — APPROVED** [review]

**Pull request merged**